pages/auth.css

```diff
@@ -0,0 +1,71 @@
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+* {
+	margin: 0;
+	box-sizing: border-box;
+	font-size: 100%;
+}
+html {
+	font-size: 62.5%;
+}
+main {
+	min-height: 100vh;
+	font-size: 1.6rem;
+	padding: 0 1.6rem;
+	display: grid;
+	gap: 1.6rem;
+	grid-template-columns: 1fr;
+	grid-template-rows: auto 1fr;
+}
+
+.header {
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	box-shadow: 0 5px 3px -3px var(--box-shadow);
+}
+
+.header-sub-container {
+	display: flex;
+	align-items: center;
+	gap: 0 2rem;
+}
+.search-container {
+	display: flex;
+	gap: 0;
+	background-color: var(--hover-color);
```

Comment:
Give a more descriptive name for color like `--hover-color-grey`.

Reply:
Sure I will work on css variable naming more.

Reply:
But as of now, I cannot change, as I used this same name at a lot of places. But I will keep this suggestion in mind.